src/spyglass/common/common_filter.py

```diff
@@ -461,7 +461,8 @@ def filter_data(
             frm, to = self._time_bound_check(
                 a_start, a_stop, timestamps, n_samples
             )
-
+            if frm == to:
```

Comment:
Are frm and to floats? If so, it would be better to use `np.isclose`

Reply:
They are floats, but they come from `np.searchsorted` on the same timestamps series, so the error is when they are exactly equivalent.

Can switch to be safe though